### PR TITLE
Detail XSRF attack

### PR DESCRIPTION
**Pull request by angelaLazar**
https://trello.com/c/a7kZX5Vg/701-using-the-state-parameter

### CODE REVIEW

File: articles/protocols/oauth2/oauth-state.md [hunk 1]
```diff
@@ -2,15 +2,17 @@
...
 ---
 
-# Using the State Parameter
+# The State Parameter
 
-The `state` parameter is one of the supported Auth0 [Authentication Parameters](/libraries/lock/v10/sending-authentication-parameters). This page is to help you understand how to utilize this this parameter which is useful to help mitigate [XSRF attacks](https://en.wikipedia.org/wiki/Cross-site_request_forgery) and for providing any contextual information (such as a return url) that you might need after the authentication process is finished.
+The `state` parameter is one of the supported Auth0 [Authentication Parameters](/libraries/lock/v10/sending-authentication-parameters), used to help mitigate [XSRF attacks](https://en.wikipedia.org/wiki/Cross-site_request_forgery).
+
+A XSRF attack can occur when a malicious program causes a user's web browser to perform an unwanted action on a trusted site that the user is currently authenticated. This type of attack specifically target state-changing requests to initiate a type of action instead of getting user data because the attacker has no way to see the response of the forged request.
```
Comment (mpaktiti) [inline articles/protocols/oauth2/oauth-state.md:11]: I think this article needs some more work. The purpose of the card is to outline the attach scenario but we only give a definition and info on how to use the param. I think that we need a sub-paragraph describing a simple attack scenario, accompanied by a diagram.

**Review by mpaktiti — CHANGES_REQUESTED** [review]
Add an attach scenario to the article

File: articles/protocols/oauth2/oauth-state.md (hunk 1 above)
Comment (angelaLazar) [reply articles/protocols/oauth2/oauth-state.md:11]: @mpaktiti A diagram is a good idea. Do you know what program can I use to create one?

Comment (mpaktiti) [reply articles/protocols/oauth2/oauth-state.md:11]: Just draw it with whatever suits you best and send it to Julieta from design team. She will make it pretty and send it back to you. Personally sometimes I just draw it on paper and take a photo. Or you could use something like this https://www.draw.io/

**Comment by angelaLazar** [comment]
@mpaktiti I have added in the diagram from design

**Pull request merged**